### ISSUE TITLE
Issue with Atmosphere when internal memory and SD card are full

### ISSUE THREAD

**Issue by andersonbisconsin** [Atmosphere-NX/Atmosphere#2090]
Description:
I found a bug in Atmosphere when opening the system. The problem seems to occur when the internal memory and SD card are full. When this happens, the system fails to start up and remains stuck at the initial Atmosphere logo.

Steps to reproduce:

Fill up the internal memory and SD card until they are both full
Attempt to start the Atmosphere system
The system fails to start and remains stuck at the initial Atmosphere logo
Expected behavior:
The system should be able to start up, regardless of the amount of memory being used.

Actual behavior:
The system fails to start up and remains stuck at the initial Atmosphere logo when the internal memory and SD card are full.

Environment:

Atmosphere version: 1.5.1
Switch firmware version: 16

**Comment by SciresM** [comment]
Have you considered not having a full SD card? I doubt that's actually atmosphere and not...a Nintendo component atmosphere is relying on.

**Comment by andersonbisconsin** [comment]
I did a test without filling up the memory and the problem still occurred. It might be a problem with my SD card, but I found it strange that the console became very slow before the problem happened. It seems like some processing is being done and then the system stops booting, it could be some sort of system rearrangement.

Is there a way to see what is on the Emunand internal memory on the PC?
